src/pages/Schedule.tsx

```
@@ -0,0 +1,153 @@
+  const time = Array.from({ length: 24 }, (_, i) => i);
+
+  const weekDates = Array.from({ length: 7 }, (_, i) => ({
+    day: days[(getDay + i) % 7],
+    num: sDay + i,
+  }));
+
+  const moveNextWeek = () => {};
+  const movePrevWeek = () => {};
+  const moveThisWeek = () => {};
+
+  const addTask = () => {
```

ESLint

[error] 'addTask' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)